quickstart/prerequisites.md

```diff
@@ -5,7 +5,7 @@
 All worker nodes must satisfy the following requirements:
 
 * **x86-64** CPU cores with SSE4.2 instruction support
-* Linux kernel **5.13 or higher** with the following modules loaded:
+* _(Tested and Recommended)_ Linux kernel **5.15 or higher** with the following modules loaded:
```

Comment:
Should we give a message of the sort "Recommended 5.13 or higher and tested with 5.15" ?
@blaisedias @tiagolobocastro, your thoughts?

Reply:
Yeah makes sense to me.